cwd/app.js

```diff
@@ -49,20 +49,19 @@
           var index = arr.indexOf(document.getElementById(this.id));
 
           arr[index].style.display = "none";
+
           if (index != arr.length - 1) {
             arr[index+1].style.display = "block";
             renderView(views[index+1]);
-
+            if(window.location.hash)
+              window.location.hash = views[index+1].name;
           } else {
             arr[0].style.display = "block";
```

Comment:
Seems worrisome to assume that our list of views always has at least one element.  We want this to be hugely customizable.

Reply:
Discussed this in person and decided not to worry about because this is something that will be handled by Oliver on https://trello.com/c/yQL9ySlr/9-handle-when-the-citywide-dashboard-database-is-empty.